src/domain/usecases/add-account.ts

```diff
@@ -9,3 +9,5 @@ export interface AddAccountModel{
 export interface AddAccount {
   add: (account: AddAccountModel) => Promise<AccountModel>
 }
+
+// just adding a comment here
```

Comment:
we should avoid making comments on source code. your code needs to be readable enough to doesnt need comments